### PR TITLE
PLAT-1059: Redis 6.2 is required for Enterprise

### PR DESCRIPTION
**Pull request by jason-seqera**
- https://seqera.atlassian.net/browse/PLAT-1059

~~Looks like this became required as far back as 23.3.~~

We're using 6.2 for 24.2+.

cc @llewellyn-sl

### CODE REVIEW

**Comment by pditommaso** [comment]
What's the reference for this change?

**Comment by jason-seqera** [comment]
@pditommaso https://seqera.atlassian.net/browse/PLAT-1058

**Comment by endre-seqera** [comment]
Added some context and reasoning to the task: https://seqera.atlassian.net/browse/PLAT-1058?focusedCommentId=30478

With the change introduced on Aug 21, 2024 (`JedisMemStoreServiceV2Impl`) - we inadvertedly introduced a breaking change, as this implementation uses commands that need at least REDIS 6.2.0

So either we should reflect his minimum REDIS 6.2.0 requirement in the docs - or change the implementation and use only commands available on REDIS 6.0

File: platform_versioned_docs/version-23.3/enterprise/configuration/overview.mdx [hunk 1]
```diff
@@ -132,7 +132,7 @@ Replace `{prefix}` in each configuration path with `/config/<application_name>`,
 Configuration values that control Seqera's interaction with databases and Redis instances. `TOWER_DB_USER`, `TOWER_DB_PASSWORD`, and `TOWER_DB_URL` must be specified using environment variables during initial Seqera Enterprise deployment in a new environment. A new installation will fail if DB values are only defined in `tower.yml` or the AWS Parameter Store. Once the database has been created, these values can be added to `tower.yml` or [AWS Parameter Store](./aws_parameter_store.mdx) entries and removed from your environment variables.
 
 :::note
-From Seqera Enterprise version 23.3, Redis version 7 is officially supported. Follow your cloud provider specifications to upgrade your instance.
+From Seqera Enterprise version 23.3, Redis version 7 is required. Follow your cloud provider specifications to upgrade your instance.
```
Comment (endre-seqera) [inline platform_versioned_docs/version-23.3/enterprise/configuration/overview.mdx:135]: As @gwright99 correctly pointed out the change should be something like this:

```
From Seqera Enterprise version 23.3, Redis version 7 is officially supported. 
From Seqera Enterprise version 24.2, at least Redis version 6.2.0 is required. 
Follow your cloud provider specifications to upgrade your instance
```

**Comment by jason-seqera** [comment]
So this seems TBD.

**Comment by jason-seqera** [comment]
So 6.2 is required for 24.2.

File: platform_versioned_docs/version-24.2/enterprise/configuration/overview.mdx [hunk 2]
```diff
@@ -132,7 +132,12 @@ Replace `{prefix}` in each configuration path with `/config/<application_name>`,
 Configuration values that control Seqera's interaction with databases and Redis instances. `TOWER_DB_USER`, `TOWER_DB_PASSWORD`, and `TOWER_DB_URL` must be specified using environment variables during initial Seqera Enterprise deployment in a new environment. A new installation will fail if DB values are only defined in `tower.yml` or the AWS Parameter Store. Once the database has been created, these values can be added to `tower.yml` or [AWS Parameter Store](./aws_parameter_store.mdx) entries and removed from your environment variables.
 
 :::note
-From Seqera Enterprise version 23.3, Redis version 7 is officially supported. Follow your cloud provider specifications to upgrade your instance.
+From Seqera Enterprise version 24.2:
+
+- Redis version 6.2 is required.
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.2/enterprise/configuration/overview.mdx:137]:
```suggestion
- Redis version 6.2 or greater is required.
```

**Review by llewellyn-sl — APPROVED** [review]
Adding "or greater". Otherwise LGTM

**Review by endre-seqera — APPROVED** [review]

**Pull request merged**